tools/pdf_util.py

```
@@ -0,0 +1,131 @@
+    return titles
+from pdfminer.high_level import extract_text
+def get_section_title(title):
+    pass
+def extract_keyword_from_pdf(fpth):
+    titles = get_subtitles(fpth)
+    print(titles)
+    text = extract_text(fpth)
+    start_index = text.lower().find('abstract')
+    end_index = text.lower().replace('\n', '').replace(' ', '').find(titles['level1'][0]['title'].replace(' ', ''))
+
+    abstract_text = text[start_index:end_index]
```

Ruff (0.8.2)

Local variable `abstract_text` is assigned to but never used

Remove assignment to unused variable `abstract_text`

(F841)